### PR TITLE
Fix js-yaml, type imports

### PR DESCRIPTION
**Pull request by jakebailey**
This fixes the use of `js-yaml` by `npm i`-ing to fix the lock, adding the right types package, and then calling the right function.

I also made a minor loader fix; the imports at the top to get the type for the lazily-loaded modules will actually cause an import to occur immediately. I can revert this part if you'd like.

There are some other type errors to do with the thrown errors, but they don't appear when running `tsc` because the version of TypeScript this repo uses is super out of date (far past what DefinitelyTyped supports, so I would strongly suggest upgrading).

### CODE REVIEW

**Review by davidtheclark — APPROVED** [review]
Thanks for fixing my facepalmworthy mistake! When CI passes, we're good to merge, I believe.

**Comment by SimenB** [comment]
Will this get a release, or is this blocked by waiting for a new maintainer (#277)?

**Pull request merged**
